spec/javascripts/helpers/test-coverage.mjs

```diff
@@ -0,0 +1,9 @@
+window.__coverage__ ??= {};
```

Comment:
Could we add a code comment here at the top of the file to say:
`This file runs in the browser`

Reply:
Done.